packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -473,11 +473,15 @@ export class ConversationAPI {
       url: `${ConversationAPI.URL.CONVERSATIONS}/${conversationId}/${ConversationAPI.URL.OTR}/${ConversationAPI.URL.MESSAGES}`,
     };
 
-    const response =
-      typeof messageData.recipients === 'object'
-        ? await this.client.sendJSON<ClientMismatch>(config, true)
-        : await this.client.sendProtocolBuffer<ClientMismatch>(config, true);
-    return response.data;
+    try {
+      const response =
+        typeof messageData.recipients === 'object'
+          ? await this.client.sendJSON<ClientMismatch>(config, true)
+          : await this.client.sendProtocolBuffer<ClientMismatch>(config, true);
+      return response.data;
+    } catch (error) {
+      throw error;
```

Comment:
What's the advantage of throwing the error after catching it?

Reply:
That the outside caller get's to see this error. When not doing it I was not able to try-catch it from the outside.